app/src/main/java/com/restaurantroulette/dto/Country.kt

```diff
@@ -3,7 +3,4 @@ package com.restaurantroulette.dto
 import com.google.gson.annotations.SerializedName
 
 data class Country(@SerializedName("countries") var code : String) {
-    override fun toString(): String {
```

Comment:
I'd leave these in.  It will return just a string, without any extra formatting that the default toString might give.